src/serlio/scripts/AEserlioTemplate.mel

```diff
@@ -101,20 +115,91 @@ global proc prtFileBrowseReplaceRPK(string $attr, string $varname, string $filte
 	}
 }
 
-global proc prtColorChooser(string $attr, string $varname) {
+global proc prtUpdateEditHighlights(string $attr){
+	string $node = plugNode($attr);
+	string $attributeName = plugAttr($attr);
+	if (!`attributeExists $attributeName $node`)
+		return;
+
+	int $userSet = `getAttr ($attr + "_user_set")`;
+	string $control = prtControlName($attr);
+
+	symbolButton -edit -visible $userSet ($control + "_user_set");
+
+	if ($userSet){
+		rowColumnLayout -edit -bgc 0.32 0.32 0.32 ($control + "_column_layout");
+		symbolButton -edit -visible true ($control + "_force_default_reset");
```

Comment:
maybe extract into function to avoid duplication with below?

Reply:
Did some cleanup by adding helper functions

Reply:
Cleaned up a bit.